int32 motor_index
float32 acceleration
float32 velocity
float32 position
bool engage
bool reset_pose
